### PR TITLE
EASY-1960 - Updated the test resources

### PR DESCRIPTION
**Pull request by rkruithof**
fixes EASY-1960

- [x] confirm invalid-doi error message (somehow it returns a 503 Service Temporarily Unavailable, instead of a 400 Bad Request now | Turns out temporarily really was temporarily)
- [x] add test bags with different doi configurations
- [x] add Readme.md to Resources directory
#### When applied it will
*  remove the DOI from the dataset.xml's
*  add a multisurface example bag, add an invalid-doi example bag
*  update the manifests

#### Where should the reviewer @DANS-KNAW/easy start?
in `~/git/service/easy/easy-sword2-dans-examples`
run `./run.sh Simple http://deasy.dans.knaw.nl/sword2/collection/1 user001 user001 src/main/resources/invalid-examples/two-available-dates`
or `./run.sh SequenceContinued http://deasy.dans.knaw.nl/sword2/collection/1 user001 user001 500000 src/main/resources/examples/revision01 src/main/resources/examples/revision02 src/main/resources/examples/revision03`

#### related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     |

### CODE REVIEW

**Comment by rkruithof** [comment]
I looked at both issues in the project, the mismatch was an easy fix, and the missing close didn't seem to be an issue for me. Regardless i moved the streams inside a try block, which may prevent any further issues on other operating systems or when the method is aborted unexpectedly(?). (Looked at this together with Stephan)

**Comment by rvanheest** [comment]
I see you committed a `src/main/resources/.DS_Store` file. Can you (1) remove it again and (2) add it to the `.gitignore` file; (2a) have a look at [the `.gitignore` in `easy-sword2`](https://github.com/DANS-KNAW/easy-sword2/blob/e7dd006e453e8b1f15c8d68baa8841dda3a7524b/.gitignore#L16) and sync them as much as possible.

File: src/main/java/nl/knaw/dans/easy/sword2examples/SimpleDeposit.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
      * Sends a bag to the easy-sword2 service and tracks its status until it is archived or failure is reported.
      * 
      * @param args
-     *        0. zipped bag to send, 1. collection URL (Col-IRI), 2. EASY user name, 3. EASY password
+     *        1. collection URL (Col-IRI), 2. EASY user name, 3. EASY password, 0. zipped bag to send,
```
Comment (rvanheest) [inline src/main/java/nl/knaw/dans/easy/sword2examples/SimpleDeposit.java]: did the order of the arguments change, or is this an accidental change?

Comment (rkruithof) [reply src/main/java/nl/knaw/dans/easy/sword2examples/SimpleDeposit.java]: The order in the block comment did not match the order in which arguments were actually parsed. 
This was mentioned in issue: https://github.com/DANS-KNAW/easy-sword2-dans-examples/issues/16
I will add a small modification to this, the current order of args should remain. the same, but the numbers should run from 0-3.

**Comment by rkruithof** [comment]
The current test-resources are laid out as follows: 

resources
├── agreement-and-default-flow
│   ├── invalid
│   │   ├── invalid-flow
│   │   ├── no-available-date
│   │   └── two-available-dates
│   └── valid
│       ├── audiences
│       ├── embargoed
│       ├── file-accessibilities
│       ├── multisurface
│       ├── revision01
│       ├── revision02
│       └── revision03
└── noaccess-flow
    ├── invalid
    │   ├── invalid-flow
    │   ├── no-available-date
    │   └── two-available-dates
    └── valid
        ├── audiences
        ├── embargoed
        ├── file-accessibilities
        ├── multisurface
        ├── revision01
        ├── revision02
        └── revision03

**Pull request merged**